tox.ini

```diff
@@ -26,17 +26,17 @@ commands = py.test --cov=etcd3 tests/
 whitelist_externals = sed
 deps = grpcio-tools
 commands =
-    sed -i -e '/gogoproto/d' etcd3/proto/rpc.proto
-    sed -i -e 's/etcd\/mvcc\/mvccpb\/kv.proto/kv.proto/g' etcd3/proto/rpc.proto
-    sed -i -e 's/etcd\/auth\/authpb\/auth.proto/auth.proto/g' etcd3/proto/rpc.proto
-    sed -i -e '/google\/api\/annotations.proto/d' etcd3/proto/rpc.proto
-    sed -i -e '/option (google.api.http)/,+3d' etcd3/proto/rpc.proto
+    gsed -i -e '/gogoproto/d' etcd3/proto/rpc.proto
+    gsed -i -e 's/etcd\/mvcc\/mvccpb\/kv.proto/kv.proto/g' etcd3/proto/rpc.proto
+    gsed -i -e 's/etcd\/auth\/authpb\/auth.proto/auth.proto/g' etcd3/proto/rpc.proto
+    gsed -i -e '/google\/api\/annotations.proto/d' etcd3/proto/rpc.proto
+    gsed -i -e '/option (google.api.http)/,+3d' etcd3/proto/rpc.proto
```

Comment:
Could we remove the gnuisms here instead? Otherwise people have to add an alias on linux machines.